services/node_create.proto

```diff
@@ -125,4 +127,12 @@ message NodeCreateTransactionBody {
      * This field is OPTIONAL.
      */
     bytes grpc_certificate_hash = 6;
+
+    /**
+     * The administrative key for the node address book entry.
+     * <p>
+     * This field is REQUIRED. It MUST be a valid key. Setting this to an invalid key or an empty `KeyList` will fail the transaction.<br/>
+     * This key MUST sign this transaction and each transaction to update this node.<br/>
```

Comment:
The former wording works better, in my opinion.
This violates the line length for the style recommendations, and also is less specification (it reads much closer to the "documentation" style we used to write, that all needs rewriting) and more just description.
```suggestion
     * An administrative key controlled by the node operator.
     * <p>
     * This key MUST sign this transaction.<br/>
     * This key MUST sign each transaction to update this node.<br/>
     * This field is REQUIRED and MUST NOT be set to an empty `KeyList`.
```

Reply:
Updated.